### ISSUE TITLE
Replicating the procedure

### ISSUE THREAD

**Issue by impredicative** [Coderx7/SimpleNet_Pytorch#2]
In February you had noted a number of procedural issues at https://github.com/EricAlcaide/SimpleNet-Keras/issues/2 . Have these all been addressed in this pytorch implementation? I assume so, but I am just being sure. Thanks!

**Comment by Coderx7** [comment]
@impredicative yes, I tried my best to replicate what I had used in Caffe, but please note that there are several differences between Caffe and Pytorch (and needless to say other frameworks) that makes you to finetune some parameters in your target framework. an example here could be the weight decay and drp ratio. I noticed setting the same weight decay and drpout ratio in pytorch results in underfitting while in Caffe those are just right values to fight overfitting. so the in the Pytorch implementation, lower values are used. the optimization policy is almost the same, with the exception that in Caffe we also use momentum with Adadelta, but such feature is not available here and the vanilla Adadelta is being used.
